CLI 2.0 release notes (#1324)
aziontech/docs: src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx

---
import Tag from 'primevue/tag'

## October 9, 2024

### Azion CLI

**Version 2.0**

> The latest version of **Azion CLI** introduces synchronized deploy: when you run the command `azion deploy`, you will be redirected to the Azion Console, where you can monitor the deployment logs for your application in real-time.

Reviewer: ```suggestion
The latest version of **Azion CLI** introduces synchronized deployment. By running the `azion deploy` command, you'll be redirected to the Azion Console, where you can monitor your application's deployment logs in real time.
```
